Document Chrome-specific limitations for description and screenshots (#10325)
GoogleChrome/web.dev: src/site/content/en/progressive-web-apps/add-manifest/index.md

The `description` property describes the purpose of your app.

In Chrome, the maximum description length is limited to 300 characters on all

> platforms. If description goes longer than that, it will be truncated and

Reviewer: ```suggestion
platforms. If the description is longer than that, it will be truncated with
```

> ellipsis appended. On Android, there is an additional limit of at most 7 lines.

Reviewer: ```suggestion
an ellipsis character. On Android, there is an additional limit of at most 7 lines.
```

  dimension.
* All screenshots matching the appropriate form factor must have the same
  aspect ratio.

> * As of Chromium 109, only screenshots with the `form_factor` set to `"wide"`

Reviewer: ```suggestion
* As of Chrome&nbsp;109, only screenshots with the `form_factor` set to `"wide"`
```

> are displayed on the desktop.

Reviewer: ```suggestion
are displayed on desktop.
```

> * As of Chromium 109, screenshots with the `form_factor` set to `"wide"` are

Reviewer: ```suggestion
* As of Chrome&nbsp;109, screenshots with the `form_factor` set to `"wide"` are
```

ignored on Android. Screenshots without `form_factor` are still displayed for

> backwards compatibility. In the future this will be changed to only display
> screenshots with the `form_factor` set to `"narrow"`.

Reviewer: ```suggestion
backwards compatibility.
```

Reviewer: Let's only document this once it happens.
